memoized_mesh.txt

```diff
@@ -1 +1 @@
+{"Vocabulary, Controlled"=>"https://id.nlm.nih.gov/mesh/D018875", "Bile Duct Neoplasms"=>"https://id.nlm.nih.gov/mesh/D001650", "Burkitt Lymphoma--etiology"=>"https://id.nlm.nih.gov/mesh/D002051Q000209"}
```

Comment:
What's here is from the specs. At some point I need to revisit mocking out the memoized read and memoized write calls, but for now I'm going to leave them alone since will unblock other work.